docs/api/ror-internal-api-swagger.yaml

```diff
@@ -612,6 +614,18 @@ components:
           type: string
           description: Human readable message
 
+    AuthMockUpdateFailedResponse:
```

Comment:
how does it affect the API version?

Reply:
In the previous MR, I increased the major, so in this MR API version change is not required